### DISCUSSION TITLE
Webspace installation 5.3 with Installatron easyApp

### DISCUSSION THREAD

**Discussion by Thaui57** [LycheeOrg/Lychee#2454] [Q&A]
I am using the installer from my ISP. The 5.3 version will be installed without error but the installation is not working without errors. Going back to version 4.13 everything works like shown in the Demo version on the Lychee homepage.

**Comment by d7415** [comment]
I don't know what you expect us to do with that.

**Reply by Thaui57** [reply]
I expect a running application on a Webspace. I installed Lychee via the Installatron page without success and via lychee.zip also without success. All diagnostic checks are green but the installation is not showing the thumbnails.

**Reply by Thaui57** [reply]
Also the Webspace hoster is trying to get the system running, also without success.

**Reply by ildyria** [reply]
> the installation is not showing the thumbnails.

Seems like you need to fix your `APP_DIR` and `APP_URL`.
Also it seems you managed to get something running but are missing thumbnails.
It would help if you could give us the content of the diagnostic page.

**Reply by Thaui57** [reply]
Everything failed to get the newest version installed on my domain discount. I gave my provider access to my account and ask for support. They also gave up installing 5.3. The version 4.13 installed with one click installation from the provider page runs well. See dji-content.de. How can I upgrade manually to version 5.5 not losing the existing installation. I followed the installation guide for the GitHub page without success. It can not be so difficult but there must be some secret. Any help is welcome.
